doc/rendszerterv.md

```diff
@@ -0,0 +1,73 @@
+---
+## 2. Projektterv
+
+---
+## 3. Üzleti folyamatok modellje
+
+---
+## 4. Követelmények
+
+---
+## 5. Funkcionális terv
+
+---
+## 6. Fizikai környezet
+
+---
+## 7. Absztrakt domain modell
+* Fogalmak:
+
+    * HTML: A HTML (angolul: HyperText Markup Language) egy leíró nyelv, melyet weboldalak készítéséhez fejlesztettek ki, és mára már internetes szabvánnyá vált a W3C támogatásával. HTML-re szükségünk lesz az alkalmazásunk frontend fejlesztése során.
+    * CSS: A CSS (Cascading Style Sheets) egy stílusleíró nyelv, mely a HTML vagy XHTML típusú strukturált dokumentumok megjelenését írja le. Weblapunk stílusát ezzel kell elkészíteni.
+    * JavaScript: A JavaScript programozási nyelv egy objektumorientált, prototípus-alapú szkriptnyelv, amelyet weboldalakon elterjedten használnak. A weblapunk programozásához igen fontos.
+
+
+---
+## 8. Architekturális terv
+
+---
+## 9. Adatbázis terv
+
+---
+## 10. Implementációs terv
+
+---
+## 11. Tesztterv
+* Egy alkalmazás készítésekor nagyon fontos szerepet töltenek be a teszttervek.
+A teszttervek segítségével tudunk különféle funkciók helyes működéséről meggyőződni, és a különféle üzleti szolgáltatások hitelesítéséről is.
+A szoftver kiadása előtt 2 tesztet kell végrehajtani, melyek az alpha illetve beta tesztek.
+
+* Alpha teszt
+    * Fejlesztő csapat fogja elvégezni.
+    * Ezen teszt során azt vizsgáljuk, hogy az alkalmazás hogyan reagál különboző böngészői környezetekben, és az adott funkciók működőképesek-e.
+    * Ha azt látjuk, hogy az előző feltételek megfelelnek, akkor sikeres volt az alpha teszt, és jöhet majd a beta teszt.
+
+* Beta teszt
+    * A beta tesztet a fejlesztő csapattól független, kívülálló személyek fogják elvégezni.
+    * Ennek a tesztnek az a célja, hogy a felhasználóktól visszajelzést kapjanak a fejlesztők az alkalmazás működésével kapcsolatban.
+
+* Amennyiben hibás működésbe ütköznek a felhasználók, akkor egy tesztelési naplóban felvezetik a tapasztalataikat, és azt visszaküldik a fejlesztőknek, a fejlesztők pedig megoldják a hibát.
+
+* Tesztelésre használt eszközök
+Böngészők: Google Chrome , Mozilla Firefox, Microsoft Edge  Operációs rendszer: Windows 10 
+
+---
+## 12. Telepítési terv
+
+---
+## 13. Karbantartási terv
+* Az alkalmazás folyamatos üzemeltetése és karbantartása, mely
+magában foglalja a programhibák kijavítását, a belső igények változása miatti
+módosításokat, valamint a környezeti feltételek változása miatt
+megfogalmazott program-, és állomány módosítási igényeket is.
+A szoftveren havonta szeretnénk karbantartásokat végezni, ezen felül bármilyen felhasználói hibajelentés után azonnali helyreállítás jön. A szoftveren évente nagyobb frissítések, módosítások fordulhatnak elő. Ellenőrizni
+kell, hogy a jövőben kiadott Android verziókkal kompatibilis-e a program.
```

Comment:
Itt mint fentebb említettem is, nem lesz külön Android és IOS, csak webes felület